boards/risc-v/bl602/bl602evb/src/bl602_bringup.c

```diff
@@ -82,14 +90,14 @@ int bl602_bringup(void)
   ret = bl602_timer_initialize("/dev/timer1", 1);
   if (ret < 0)
     {
-      syslog(LOG_ERR, "Failed to initialize /dev/timer1 Driver: %d\n", ret);
+      ferr("Failed to initialize /dev/timer1 Driver: %d\n", ret);
       return ret;
     }
 #elif defined(CONFIG_BL602_TIMER1) && defined(CONFIG_ONESHOT)
   os = oneshot_initialize(1, 1);
   if (os == NULL)
     {
-      syslog(LOG_ERR, "ERROR: oneshot_initialize failed\n");
+      ferr("ERROR: oneshot_initialize failed\n");
```

Comment:
f[err,warn,info] are the filesystem debug helpers.   There are ones for timer, pwm, etc You can find them enumerated here `nuttx/include/debug.h`.  There are even some nice helpers for doing things like dumping buffers which can be nice at the debug level.  That said in this file we should be just using the syslog.